lib/shoulda/matchers/active_record/have_db_index_matcher.rb

```diff
@@ -24,6 +24,29 @@ module ActiveRecord
       #       should have_db_index(:user_id)
       #     end
       #
+      # has an index and an array of columns.
```

Comment:
This should be a full sentence, like this:
```suggestion
      # It can also be used to test that the table that backs your model
      # has an index on an array of specific columns.
```